system/include/libc/assert.h

```diff
@@ -16,6 +16,9 @@
 extern "C" {
 #endif
 
+#if __EMSCRIPTEN__
```

Comment:
Should this be done across the whole project in this PR, or just this one in libc for now?